src/Microsoft.AspNetCore.Razor.Language/Legacy/SpanContext.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.AspNetCore.Razor.Language.Syntax.InternalSyntax;
+
+namespace Microsoft.AspNetCore.Razor.Language.Legacy
+{
+    internal class SpanContext
```

Comment:
>SpanContext [](start = 19, length = 11)

I see that this is in the .Leygacy namespace which is OK. I'm hoping that this is somewhat of a temporary bridge between old code and new code.

Reply:
Yes but they won't go away anytime soon because we don't have a plan for getting rid of EditHandlers

---
In reply to: [233617210](https://github.com/aspnet/Razor/pull/2590#discussion_r233617210) [](ancestors = 233617210)